context/app/static/js/pages/Vis/Vis.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { useContext } from 'react';
+import Paper from '@material-ui/core/Paper';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+
+import { StyledTableContainer, HeaderCell } from 'js/shared-styles/Table';
+import { AppContext } from 'js/components/Providers';
+import useSearchData from 'js/hooks/useSearchData';
+
+/* JSON query
+ {
+     "size": 0,
+     "query": {
+         "bool": {
+             "filter": {
+                 "term": {
+                     "entity_type": "donor"
+                 }
+             }
+         }
+     },
+     "aggs": {
+         "mapped_metadata.sex": {
+             "terms": {
+                 "field": "mapped_metadata.sex.keyword"
+             }
+         }
+     }
+ }
+
+ */
+
+const donorSexQuery = {
```

Comment:
Either race or sex is a fine example to start with, but make sure the variable name matches the content.